ubuntudesign/canonical-vanilla-theme
# Allowed theme building to be viewed via demo

# Done
- Added a build file to wrap the theme
- Updated demo to use the new generated styles
- Renamed theme.scss to _theme.scss so its not complied on `gulp build`

## QA
- Run `rm -rf build node_modules`
- Run `npm install`
- Run `google-chome demo/index.html` and check it is unstyled
- Run `gulp build`
- Refresh the demo page and see the new styling
